### ISSUE TITLE
Thumbnail check issue

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#256]
Looks like we could use some error handling in the Thumbnail check.  Just so happened to catch mine today with a ~42000 queue error out due to a truncated image.  Based on previous log observations, I'm going to assume it'll always error here, leaving the thousands in the queue.

```
[2022-05-31 17:38:49,538: ERROR/ForkPoolWorker-32] Task thumbnail_check[4886b7a7-52d6-4857-b00f-e6ddf0042cc3] raised unexpected: OSError('image file is truncated (83 bytes not processed)')

Traceback (most recent call last):
File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
R = retval = fun(*args, **kwargs)
File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
return self.run(*args, **kwargs)
File "/app/home/tasks.py", line 212, in thumbnail_check
validate_thumbnails()
File "/app/home/src/download/thumbnails.py", line 343, in validate_thumbnails
handler.download_vid(thumbs_to_download)
File "/app/home/src/download/thumbnails.py", line 182, in download_vid
img_raw = img_raw.crop((0, offset, width, height - offset))
File "/root/.local/lib/python3.10/site-packages/PIL/Image.py", line 1208, in crop
self.load()
File "/root/.local/lib/python3.10/site-packages/PIL/ImageFile.py", line 251, in load
raise OSError(
OSError: image file is truncated (83 bytes not processed)
```

**Comment by bbilly1** [comment]
A filesystem rescan from the settings page will also trigger a thumbnail check.

**Comment by bbilly1** [comment]
Fix is now in v0.1.6, does that solve the problem? If yes, please close the issue, else let me know.
